src/index.ts

```diff
@@ -86,8 +85,23 @@ export class OAuth2VueClient {
     this.client = null
   }
 
-  public async login(username: string, password: string): Promise<void> {
-    const client = await this.oAuthFactory.login(username, password)
+  public async loginPassword(username: string, password: string): Promise<void> {
+    if (this.options.isMock)
```

Comment:
if'ke zonder brackets :)

Reply:
da is die stomme linter die dat automatisch weghaalt